nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -1,6 +1,6 @@
 """
 Transformer for computing seeds signals.
-=======
+-------
```

Comment:
That title line was too short anyhow, so it's gone from invalid to invalid; I'll fix it myself after merge.